tests/ha/test_ha_node_health.py

```diff
@@ -608,89 +497,68 @@ def test_single_node_multiple_times_unsafe_shutdown(self):
+        LOGGER.info("Started: Test to check single node status with multiple unsafe shutdown.")
 
         LOGGER.info("Get the node for multiple unsafe shutdown.")
         node_index = self.system_random.choice(range(self.num_nodes))
 
-        LOGGER.info(
-            "Shutdown %s node multiple time and check status.",
-            self.srvnode_list[node_index])
+        LOGGER.info("Shutdown %s node multiple time and check status.",
+                    self.srvnode_list[node_index])
         for loop in range(self.loop_count):
             self.restored = False
-            LOGGER.info(
-                "Shutting down node: %s, Loop: %s",
-                self.srvnode_list[node_index],
-                loop)
+            LOGGER.info("Shutting down node: %s, Loop: %s", self.srvnode_list[node_index], loop)
             if self.setup_type == "HW":
-                LOGGER.debug(
-                    "HW: Need to disable stonith on the node before shutdown")
+                LOGGER.debug("HW: Need to disable stonith on the node before shutdown")
                 self.node_list[node_index].execute_cmd(
                     common_cmds.PCS_RESOURCE_STONITH_CMD.format("disable", node_index+1),
                     read_lines=True)
-
-            resp = self.ha_obj.host_safe_unsafe_power_off(
-                host=self.host_list[node_index],
-                bmc_obj=self.bmc_list[node_index],
-                node_obj=self.node_list[node_index])
-            assert_utils.assert_true(
-                resp, f"{self.host_list[node_index]} has not shutdown yet.")
-            LOGGER.info("%s is powered off.", self.host_list[node_index])
+            resp = self.ha_obj.host_safe_unsafe_power_off(host=self.host_list[node_index],
+                                                          bmc_obj=self.bmc_list[node_index],
+                                                          node_obj=self.node_list[node_index])
+            assert_utils.assert_true(resp, f"{self.host_list[node_index]} has not shutdown yet.")
 
             LOGGER.info("Get the new node on which CSM service failover.")
             if self.srvnode_list[node_index] == self.srvnode_list[-1]:
                 nd_obj = self.node_list[0]
             else:
                 nd_obj = self.node_list[node_index + 1]
-            resp = self.ha_obj.check_csm_service(
-                nd_obj, self.srvnode_list, self.sys_list)
+            resp = self.ha_obj.check_csm_service(nd_obj, self.srvnode_list, self.sys_list)
             assert_utils.assert_true(resp[0], resp[1])
             sys_obj = resp[1]
 
             check_rem_node = [
-                "failed" if num == node_index else "online" for num in range(
-                    self.num_nodes)]
-            resp = self.ha_obj.verify_node_health_status(
-                sys_obj, status=check_rem_node)
+                "failed" if num == node_index else "online" for num in range(self.num_nodes)]
```

Comment:
plz check indent

Reply:
Ran pylint on this file manually.. Looks good to me